_opensearch/install/plugins.md

```diff
@@ -52,6 +52,29 @@ opensearch-security
 opensearch-sql
 ```
 
+You can also list plugins that are installed by using the [CAT API]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-plugins/):
+
+#### Path and HTTP method
```

Comment:
I think this should be a level-3 heading.

Reply:
for our style guide we actually keep these as L4s